test/app.test.js

```diff
@@ -45,6 +45,58 @@ describe('simple app', function() {
 			.expect(200, 'Static file\n', done);
 	});
 
+	describe('backend access', function () {
+		before(function () {
+			process.env.NODE_ENV = 'production';
+		});
+
+		after(function () {
+			process.env.NODE_ENV = '';
+		});
+
+		xit('should 401 for arbitrary route without a backend access key in production', function (done) {
+			request(app)
+				.get('/vanilla')
+				.expect('ft-backend-authentication', /false/)
+				.expect(401, done);
+		});
+
+		xit('should 401 for arbitrary route with incorrect backend access key in production', function (done) {
+			request(app)
+				.get('/vanilla')
+				.set('FT-Next-Backend-Key', 'as-if')
+				.expect('ft-backend-authentication', /false/)
+				.expect(401, done);
+		});
+
+		it('should allow robots.txt through without backend access key', function (done) {
```

Comment:
Um don't know. Could robots.txt prevent them crawling __about, __health etc... 

Happy to remove if this is nonsense